index.html

```diff
@@ -145,125 +149,364 @@ <h2>
+                </p>
+                <p>
+                  Conversely, if pointer lock is exited via
+                  {{Document/exitPointerLock()}} no [=transient activation=] is
+                  required to reenter pointer lock. This enables applications
+                  that frequently move between interaction modes, and ones that
+                  may do so based on a timer or remote network activity.
+                </p>
+                <p>
+                  Pointer lock is commonly combined with fullscreen
+                  [[FULLSCREEN]], and if an [=transient activation=] is used to
+                  enter fullscreen it is sufficient for a subsequent
+                  {{Element/requestPointerLock()}}.
+                </p>
+              </aside>
+            </li>
+            <li>If [=this=]'s [=Node/node document=]'s [=Document/active
+            sandboxing flag set=] has the [=sandboxed pointer lock browsing
+            context flag=] set:
+              <ol>
+                <li>[=Queue an element task=] on the [=user interaction task
+                source=], given this, to perform the following steps:
+                  <ol>
+                    <li>[=Fire an event=] named {{pointerlockerror}} at
+                    [=this=]'s [=Node/node document=].
+                    </li>
+                    <li>[=Reject=] <var>promise</var> with a
+                    "{{SecurityError}}" {{DOMException}}.
+                    </li>
+                  </ol>
+                </li>
+                <li>Return <var>promise</var>.
+                </li>
+              </ol>
+            </li>
+            <li>If [=options=]["{{PointerLockOptions/unadjustedMovement}}"] is
+            true and the platform does not support
+            {{PointerLockOptions/unadjustedMovement}}:
+              <ol>
+                <li>[=Queue an element task=] on the [=user interaction task
+                source=], given this, to perform the following steps:
+                  <ol>
+                    <li>[=Fire an event=] named {{pointerlockerror}} at
+                    [=this=]'s [=Node/node document=].
+                    </li>
+                    <li>[=Reject=] <var>promise</var> with a
+                    "{{NotSupportedError}}" {{DOMException}}.
+                    </li>
+                  </ol>
+                </li>
+                <li>Return <var>promise</var>.
+                </li>
+              </ol>
+              <aside class="note" data-link-for="Document">
+                "{{NotSupportedError}}" {{DOMException}} should be reserved for
+                unsupported options in future iterations of this spec. That is,
+                no other error conditions should use "{{NotSupportedError}}",
+                so that web developers can use it for feature testing.
+              </aside>
+            </li>
+            <li>If none of the above errors have occured
```

Comment:
One check we are missing is for Document's visibility state. A hidden document shouldn't be allowed to request pointer lock. 

I'll file a bug for that.

Reply:
Ah, I see the algorithm is using window "focus"... that's not correct. I needs to change to document visibility. Take a look at:

https://www.w3.org/TR/geolocation/#request-a-position

For how to do the visibility check.

Reply:
I added an <aside> after the focus check step with #93 associated. Then we can follow from the issue later.